Documentation/Development.md

```diff
@@ -154,46 +154,46 @@ For contributors on macOS who need to test on Linux, install Docker and use the
+$ Utilities/Docker/docker-utils bootstrap # will bootstrap SwiftPM on the Linux container
+$ Utilities/Docker/docker-utils run bash # to run an interactive Bash shell in the container
 $ Utilities/Docker/docker-utils swift-build # to run swift-build in the container
 $ Utilities/Docker/docker-utils swift-test # to run swift-test in the container
 $ Utilities/Docker/docker-utils swift-run # to run swift-run in the container
 ```
 
-## Using custom Swift compilers
+## Using Custom Swift Compilers
 
-SwiftPM needs Swift compiler to parse Package.swift manifest files and to
-compile Swift source files. You can use `SWIFT_EXEC` and `SWIFT_EXEC_MANIFEST`
+SwiftPM needs the Swift compiler to parse `Package.swift` manifest files and to
+compile Swift source files. You can use the `SWIFT_EXEC` and `SWIFT_EXEC_MANIFEST`
 environment variables to control which compiler to use for these operations.
 
 `SWIFT_EXEC_MANIFEST`: This variable controls which compiler to use for parsing
-Package.swift manifest files. The lookup order for the manifest compiler is:
-SWIFT_EXEC_MANIFEST, swiftc adjacent to swiftpm binaries, SWIFT_EXEC
+`Package.swift` manifest files. The lookup order for the manifest compiler is:
+`SWIFT_EXEC_MANIFEST`, `swiftc` adjacent to the `swiftpm` binaries, then `SWIFT_EXEC`
 
 `SWIFT_EXEC`: This variable controls which compiler to use for compiling Swift
-sources. The lookup order for the sources compiler is: SWIFT_EXEC, swiftc adjacent
-to swiftpm binaries. This is also useful for Swift compiler developers when they
+sources. The lookup order for the sources' compiler is: `SWIFT_EXEC`, then `swiftc` adjacent
+to `swiftpm` binaries. This is also useful for Swift compiler developers when they
 want to use a debug compiler with SwiftPM.
 
 ```sh
 $ SWIFT_EXEC=/path/to/my/built/swiftc swift build
 ```
 
-## Overriding path to the runtime libraries
+## Overriding the Path to the Runtime Libraries
 
-SwiftPM computes the path of runtime libraries relative to where it is
+SwiftPM computes the path of its runtime libraries relative to where it is
 installed. This path can be overridden by setting the environment variable
 `SWIFTPM_PD_LIBS` to a directory containing the libraries. This can be a list of
-absolute search paths separated by colon (":"). SwiftPM will choose the first
-path which exists on disk. If none of the path are present on disk, it will fall
+absolute search paths separated by colons (":"). SwiftPM will choose the first
+path which exists on disk. If none of the paths are present on disk, it will fall
 back to built-in computation.
 
 ## Skip SwiftPM tests
 
 SwiftPM has a hidden env variable `_SWIFTPM_SKIP_TESTS_LIST` that can be used
 to skip a list of tests. This value of the variable is either a file path that contains a
-new-line separated list of tests to skip or a colon separated list of tests.
+newline separated list of tests to skip, or a colon separated list of tests.
```

Comment:
I think it's `colon-separated`.

Reply:
Wikipedia and other sites seem to agree, so I changed to "colon-separated."